src/main/java/br/com/zup/concessionariacitroen/services/PedidoService.java

```diff
@@ -0,0 +1,35 @@
+package br.com.zup.concessionariacitroen.services;
+
+import br.com.zup.concessionariacitroen.models.Pedido;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PedidoService {
+
+    private static List<Pedido> listaPedidos = new ArrayList<>();
+
+    public Pedido cadastrarPedido(Pedido pedido) {
+        listaPedidos.add(pedido);
+        return pedido;
+    }
+
+    public List<Pedido> listarPedidos() {
+        return listaPedidos;
+    }
+
+    public Pedido pesquisarPedidoPeloNumero(int numeroPedido) {
+        Pedido pedidoEncontrado = null;
+
+        for (Pedido objetoPedido : listaPedidos) {
+
+            if (objetoPedido.getNumeroPedido() == numeroPedido) {
+                pedidoEncontrado = objetoPedido;
```

Comment:
Pode excluir a variável, e retornar conforme abaixo:
```suggestion
                return objetoPedido;
```